<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationPreview</class>
 <widget class="QWidget" name="AnimationPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>46</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="fpsSlider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="fpsLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS: 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actualViewButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Size</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
